Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -73,6 +73,8 @@ public class JavaScriptLifter: ComponentBase, Lifter {
         }
 
         if options.contains(.collectTypes) {
+            w.emit("function typeCollectionMain() {")
```

Comment:
Maybe this could also use a short comment about why everything is wrapped into another function (so that type collection variables don't become global variables which would make them accessible to Fuzzilli generated code)?